lib/commands/account/account-list.ts

```diff
@@ -30,7 +30,7 @@ export class AccountListCommand extends AccountCommandBase implements ICommand {
 			output = table.toString();
 		}
 
-		this.$logger.out(output);
+		this.$logger.info(output);
```

Comment:
Why is `this.$logger.out` not working anymore?

Reply:
`logger.out` is deprecated and will be removed in one of the future releases, that's why I'm replacing it with `logger.info`

Reply:
Based on your other comment, `this.$logger.info ` will print the message on the same line. I think this is going to be a problem here. Can you confirm that the table will be properly displayed using `this.$logger.info` (it won't start on the same line and it will end with new line)

Reply:
`logger.info` will print on the same line only when `{[this.$constants.LoggerConfigData.skipNewLine]: true }` is passed. Here we do not pass it, so `logger.info` will add new line at the end of the passed input (same as the way `logger.out` was working in the past.

Reply:
logger.out should have worked correctly and it will work after merging this PR: https://github.com/NativeScript/nativescript-cli/pull/4670

However, using the new API is the preferred way.